app/views/components/summaryCardLPP.scala.html

```diff
@@ -140,7 +140,7 @@ <h4 class="app-summary-card__title">
     <div class="app-summary-card__body">
     @govukSummaryList(SummaryList(summaryCard.cardRows))
     @if(!summaryCard.penaltyCategory.equals(MANUAL) && !summaryCard.isVatPaid && summaryCard.appealStatus.getOrElse(AppealStatusEnum.Unappealable) == AppealStatusEnum.Unappealable) {
-        @if(!summaryCard.showFindOutHowToAppeal || summaryCard.isCentralAssessment) {
+        @if(!summaryCard.showFindOutHowToAppeal) {
```

Comment:
Please create a separate if statement and use the FS created in https://github.com/hmrc/penalties-frontend/pull/394